sql/wsrep_mysqld.h

```diff
@@ -180,7 +180,8 @@ void wsrep_init_startup(bool before);
 void wsrep_recover_sr_from_storage(THD *);
 
 // Other wsrep global variables
-extern my_bool     wsrep_inited; // whether wsrep is initialized ?
+extern my_bool  wsrep_inited; // whether wsrep is initialized ?
+extern my_bool  wsrep_init_SST_before_SE;
```

Comment:
Seeing that `wsrep_before_SE()` and `wsrep_init_SST_before_SE` have the same visibility, what are the benefits to have the latter? The disadvantage would be that due to a bug `wsrep_init_SST_before_SE` may be used uninitialized, for example.

Reply:
I would agree though that `wsrep_before_SE()` name is confusing, so changing it to `wsrep_SST_before_SE()` would be certainly welcome!

Reply:
I will rename function as it makes more sense. 

Reasons additional variable is that, at that point we have passed several conditions, one of this condition variables are not accessible outside of mysqld.cc.

General note, I'm expecting that this implementation is going to several changes more. There is regression found on several tests and this needs to be addressed.